crates/bevy_dev_tools/src/picking_debug.rs

```diff
@@ -0,0 +1,323 @@
+//! Text and on-screen debugging tools
+
+use core::cmp::Ordering;
+use core::fmt::{Debug, Display, Formatter, Result};
+
+use bevy_app::prelude::*;
+use bevy_asset::prelude::*;
+use bevy_color::prelude::*;
+use bevy_ecs::prelude::*;
+use bevy_picking::backend::HitData;
+use bevy_picking::hover::HoverMap;
+use bevy_picking::pointer::{Location, PointerId, PointerPress};
+use bevy_picking::prelude::*;
+use bevy_picking::{pointer, PickSet};
+use bevy_reflect::prelude::*;
+use bevy_render::prelude::*;
+use bevy_text::prelude::*;
+use bevy_ui::prelude::*;
+use bevy_utils::tracing::{debug, trace};
```

Comment:
As of https://github.com/bevyengine/bevy/pull/17161, Tracing is no longer exported from `bevy_utils`.
```suggestion
use tracing::{debug, trace};
```